pkg/kv/kvserver/replica_raftstorage.go

```diff
@@ -809,31 +812,25 @@ func (r *Replica) applySnapshot(
 		// Time to ingest SSTs.
 		ingestion time.Time
 	}
-	log.Infof(ctx, "applying snapshot of type %s [id=%s index=%d]", inSnap.snapType,
-		inSnap.SnapUUID.Short(), nonemptySnap.Metadata.Index)
+	log.Infof(ctx, "applying snapshot of type %s [id=%s from=%s index=%d]", inSnap.snapType,
```

Comment:
shouldn't this ideally be something like `log.Infof(ctx, applying %s, inSnap)` (which uses `(*IncomingSnapshot).SafeFormat`?

Reply:
Notice that the `index` field here comes from `nonemptySnap`, not `inSnap`.

Reply:
All of the various index fields must be the same.

Reply:
Ok, thanks -- updated.